### ISSUE TITLE
"sass-rails" gem is deprecated (EOF: 26 March 2019)

### ISSUE THREAD

**Issue by artur-beljajev** [internetee/rest-whois#123]
https://github.com/rails/sass-rails/issues/420

**Comment by maciej-szlosarczyk** [comment]
As far as I can see we don't use it for anything, there are 0 scss files in the repo.

**Comment by artur-beljajev** [comment]
But there are `.sass` files. Aren't they require this gem? Also, if I remember correctly there was something else preventing this gem removing. Maybe `sprockets` requires it, can't recall.

**Comment by maciej-szlosarczyk** [comment]
> But there are .sass files. 

I don't see any:

![image](https://user-images.githubusercontent.com/2331765/47709712-8eb0d400-dc39-11e8-898e-03e3675e455c.png)

> Maybe sprockets requires it, can't recall.

It does not seem so, here's the `Gemfile.lock` part for sprockets:

```
    sprockets (3.7.2)
      concurrent-ruby (~> 1.0)
      rack (> 1, < 3)
    sprockets-rails (3.2.1)
      actionpack (>= 4.0)
      activesupport (>= 4.0)
      sprockets (>= 3.0.0)

**Comment by artur-beljajev** [comment]
Oop, I messed up projects. Surely it can be removed from `rest-whois`.

**Comment by maciej-szlosarczyk** [comment]
Fixed in #127
